packer_update_centos_base.json

```diff
@@ -32,6 +32,11 @@
   ],
   "provisioners" : [
     {
+      "type": "file",
+      "source": "centos6.elrepo.repo",
+      "destination": "/tmp/centos6.elrepo.repo"
```

Comment:
minor - we don't need to copy this file for centos 7

Reply:
Correct, but there's no good way in Packer to not copy it.  So we do a tiny bit of extra work so that we can have the same config file.